src/examples/dlt-example-multicast-client.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * SPDX license identifier: MPL-2.0
+ *
+ * Copyright (C) 2020, CriticalTechworks
+ *
+ * This file is part of GENIVI Project DLT - Diagnostic Log and Trace.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License (MPL), v. 2.0.
+ * If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * For further information see http://www.genivi.org/.
+ */
+
+/*!
+ * \author Joao Sousa <joao.sa.sousa@ctw.bmwgroup.com>
+ *
+ * \copyright Copyright © 2020 CriticalTechworks. \n
+ * License MPL-2.0: Mozilla Public License version 2.0 http://mozilla.org/MPL/2.0/.
+ *
+ * \file dlt-example-multicast-client.c
+ */
+#include <stdio.h>
+#include <syslog.h>
+#include <time.h>
+#include <sys/time.h>
+
+#include "dlt_client.h"
+#include "dlt_client_cfg.h"
+#include "dlt_types.h"
+
+#define MULTICAST_GROUP_ADDRESS "239.1.2.3"
```

Comment:
Can we align this address to default defined in `dlt.conf`?

Reply:
Agree